### ISSUE TITLE
Decide what accessibility testing tool we are going to use

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [100Automations/Website#420]
### Overview
We need to do accessibility  testing on the webiste to determine if changes need to be made before formal launch

### Action Items
- [x] Read the article 
- [x] Write up (in the comments below) a plan a plan for testing the site.
- [ ] Make new issues to break up the site testing

### Details
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

Here is a link to a list of issue templates that hack for la uses that have to do with accessibility and other site testing (if you use lighthouse)

### Resources
[Spreadsheet with all the findings](https://docs.google.com/spreadsheets/d/1D3ImG4TwDG-M7QZepoOaajCRWAVAIBGV_7T4KAWhgSY/edit#gid=0)
[Figma with the images](https://www.figma.com/file/LAPaFDaX2oYz0t46R7Lhyz/100-Automations--Accessibility-Test?node-id=0%3A1)

**Comment by superbunker** [comment]
I think we should do more than one tool -- they have different opinions. Maybe 2-3 people could each run a test?

Henry will research the tools -- we'll decide saturday

**Comment by gstill** [comment]
Review my [accessibility audit finding notes](https://docs.google.com/spreadsheets/d/1D3ImG4TwDG-M7QZepoOaajCRWAVAIBGV_7T4KAWhgSY/edit#gid=0
) then refind the major issues and transfer them into an excel document for the development team to access by Saturday, Oct 2nd. 

Images in personal [figma](https://www.figma.com/file/LAPaFDaX2oYz0t46R7Lhyz/100-Automations--Accessibility-Test?node-id=0%3A1)

**Comment by gstill** [comment]
Add a review of the tools stating the pros and cons of each of them in an excelsheet and create a research plan by Saturday, Oct 2nd.

**Comment by ExperimentsInHonesty** [comment]
@gstill  will update spreadsheet with category of issue and number of corresponding image in figma.

**Comment by henlatourrette** [comment]
@gstill has finished the audit with all the accessibility testing tools and all the findings are written down in the spreadsheet. We now need to define the next steps regarding the solutions.

Gillian says she needs clarification on what @ExperimentsInHonesty means by "number of corresponding image in figma."

**Comment by superbunker** [comment]
The testing is complete.
Next steps are to review and compile the errors, and create new issues by priority.
